src/fu-tool.c

```diff
@@ -5087,11 +5232,11 @@ main(int argc, char *argv[])
 			      _("List EFI variables with a specific GUID"),
 			      fu_util_efivar_list);
 	fu_util_cmd_array_add(cmd_array,
-			      "efivar-boot",
+			      "efivar-boot [next|order|create|delete]",
```

Comment:
update the bash completion for the new arguments too

Reply:
As a preference; I think that this should actually be 4 commands.  `efivar-create`, `efivar-delete`, `efivar-bootorder` and `efivar-bootnext`.  Thoughts?

Reply:
I'm completely lost for this kind of thing; can you give me a hand please?

Reply:
Okay, I can certainly create new arguments without handholding :) I do think "boot" needs to be in the name somewhere -- maybe efivar-boot-create efivar-boot-delete etc? or efiboot-create efiboot-delete etc?

Reply:
Sure!

Reply:
Okay, fixed up.